third_party/xla/xla/stream_executor/rocm/rocm_blas.cc

```diff
@@ -110,6 +110,25 @@ bool ROCMBlas::Init() {
     return false;
   }
 #endif
+
+  int dev = 0;
```

Comment:
@wenchenvincent added some routines to computeCapability that we can use to check for MFMA, or certain GPU or later. Can we use those here instead of explicitly checking inline?

See:
https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/commit/01e64afa867f94eeb348116d642c6a02e0dc9e3f

Reply:
I think, this is a good idea: one can get access to RocmComputeCapability from 'stream'